modules/partition-replicator/src/integrationTest/java/org/apache/ignite/internal/partition/replicator/ItReplicaLifecycleTest.java

```diff
@@ -656,6 +660,41 @@ public void testScanCloseReplicaRequest() throws Exception {
         assertDoesNotThrow(tx::commit);
     }
 
+    @Test
+    public void testReplicaSafeTimeSyncRequest() throws Exception {
+        // Prepare a single node cluster.
+        startCluster(2);
+        Node node0 = getNode(0);
+        List<Set<Assignment>> assignments = PartitionDistributionUtils.calculateAssignments(
+                cluster.stream().map(n -> n.name).collect(toList()), 1, 1);
+
+        List<TokenizedAssignments> tokenizedAssignments = assignments.stream()
+                .map(a -> new TokenizedAssignmentsImpl(a, Integer.MIN_VALUE))
+                .collect(toList());
+
+        placementDriver.setPrimary(node0.clusterService.topologyService().localMember());
+        placementDriver.setAssignments(tokenizedAssignments);
+
+        // Prepare a zone.
+        String zoneName = "test_zone";
+        createZone(node0, zoneName, 1, 2);
+        int zoneId = DistributionZonesTestUtil.getZoneId(node0.catalogManager, zoneName, node0.hybridClock.nowLong());
+        int partId = 0;
+
+        // Create a table to work with.
+        createTable(node0, zoneName, "test_table");
```

Comment:
Please adjust the test in a following way.
1. Create zone
2. ensure that safeTime was successfully adjusted. // It should be possible to adjust timestamp even if there's no tables in zone.
3. Create table
4. await table creation and ensure that safeTime was successfully adjusted.

Reply:
Done